src/index.ts

```diff
@@ -333,3 +333,23 @@ export const parse = (version: string | null) => {
 
 	return parsed;
 };
+
+/**
+ * @summary Check if a version string is valid
+ * @name valid
+ * @public
+ * @function
+ *
+ * @description Return the parsed version, or null if it's not valid.
+ *
+ * @param {string|null} version
```

Comment:
I guess that this should read `{string|null|undefined}`?

Reply:
yep, I'll rebase it and it'll pick up the new version Type

Reply:
Then I'll update the others to read the same